Tests/NIOHTTPClientTests/HTTPClientTestUtils.swift

```diff
@@ -15,13 +15,21 @@
 import Foundation
 import NIO
 import NIOHTTP1
-@testable import NIOHTTPClient
+import NIOHTTPClient
 import NIOSSL
 
 class TestHTTPDelegate: HTTPClientResponseDelegate {
     typealias Response = Void
 
-    var state = ResponseAccumulator.State.idle
+    enum State {
```

Comment:
`@testable` was only required because we were piggy-packing on `ResponseAccumulator.State`. As we're not using the rest of `ResponseAccumulator` I think it's better to just define the state we need here.